mantle/cmd/kola/kola.go

```diff
@@ -103,12 +114,14 @@ This can be useful for e.g. serving locally built OSTree repos to qemu.
 
 	runExternals []string
 	runMultiply  int
+	runRerunFlag bool
 )
 
 func init() {
 	root.AddCommand(cmdRun)
 	cmdRun.Flags().StringArrayVarP(&runExternals, "exttest", "E", nil, "Externally defined tests (will be found in DIR/tests/kola)")
 	cmdRun.Flags().IntVar(&runMultiply, "multiply", 0, "Run the provided tests N times (useful to find race conditions)")
+	cmdRun.Flags().BoolVar(&runRerunFlag, "rerun", false, "re-run failed tests once")
```

Comment:
I think this is OK as is, but it feels like we might as well make it something like `--retry=N` to support an arbitrary number of retries.  I can imagine 2 or 3 being more of a sweet spot, depending.